### ISSUE TITLE
🐛 [firebase_app_distribution] Add support to Firebase App Distribution

### ISSUE THREAD

**Issue by nohe427** [FirebaseExtended/flutterfire#11607]
## What feature would you like to see?

There is a feature to collect feedback from users that is not currently covered by flutterfire. Can we add coverage for this feature?

Doc links : https://firebase.google.com/docs/app-distribution/collect-feedback-from-testers

**Comment by russellwheatley** [comment]
Hey @nohe427, is there any documentation for iOS [collecting feedback from users](https://firebase.google.com/docs/app-distribution/collect-feedback-from-testers)?

**Comment by nohe427** [comment]
I don't believe anything was released for iOS in regards to collecting feedback from users

**Comment by samanson123** [comment]
Would be very useful if this feature could be implemented
